### PR TITLE
Fix bash completion(bsc#1166329)

### PR DESCRIPTION
**Pull request by liangxin1300**
And sync contents from NHF's crmsh.spec

### CODE REVIEW

File: crmsh.spec.in [hunk 1]
```diff
@@ -179,7 +160,7 @@ touch -r doc/crm.8.adoc{.timestamp,}
 make %{_smp_mflags} VERSION="%{version}" sysconfdir=%{_sysconfdir} localstatedir=%{_var}
 
 %if %{with regression_tests}
-tox -q
+./test/run --quiet
```
Comment (arbulu89) [inline crmsh.spec.in]: Why have you updated this? Wasn't better to use tox?

Comment (liangxin1300) [reply crmsh.spec.in]: Honestly I didn't notice this, just copy from NHF's spec file.
I will take a look at it. Thanks for pointing out!

Comment (liangxin1300) [reply crmsh.spec.in]: @arbulu89 , Already changed to `tox`

Comment (liangxin1300) [reply crmsh.spec.in]: If this PR is ready, I will merge it when your CI's change for `spec.in` file is ready

Comment (arbulu89) [reply crmsh.spec.in]: @liangxin1300 Perfect. I will let you know when the CI project is updated and working. Wait meanwhile.

**Comment by liangxin1300** [comment]
Thanks! @arbulu89

**Pull request merged**